tools/cldr-apps/js/src/views/MainHeader.vue

```diff
@@ -5,9 +5,9 @@
       <li>
         {{ stVersion }} {{ stPhase }}
         <span
-          class="ddlException"
-          v-if="ddlException"
-          title="Note: As a non-TC DDL locale, this phase has been extended."
+          class="extendedException"
+          v-if="extendedException"
+          title="Note: This phase has been extended for this locale."
```

Comment:
If TC locales are in Vetting mode, and DDL locales are still in Submission mode, then if a DDL locale is selected, will this show "Vetting (extended)" or "Submission (extended)"? I'm guessing the latter but not quite sure

Reply:
The latter. And it doesn't have to be a DDL locale, it could be one on the extended list.